src/graphql/queries/most-bets.js

```diff
@@ -30,10 +32,23 @@ module.exports = async (
   { db: { Bets } },
 ) => {
   const query = filter ? { $or: buildFilters(lowercaseFilters(filter)) } : {};
-  const result = await Bets.find(query);
+  let result = await Bets.find(query);
+  const { eventAddress } = filter || {};
```

Comment:
make a separate func to `buildResultSetFilters`. also can rename the original `buildFilters` -> `buildBetFilters`